### ISSUE TITLE
Frameless window has no border radius when `show` set to false

### ISSUE THREAD

**Issue by marcbachmann** [atom/electron#517]
When creating a browser-window with option `show:false` and triggering a show
on webContents load, it doesn't have a border-radius at the top.

```coffee
    loginWindow = new BrowserWindow
      title: 'Login'
      width: 450
      height: 350
      show: false
      frame: false
      resizable: false

    loginWindow.loadUrl('file://' + config.root + '/error.html')

    loginWindow.webContents.on 'did-finish-load', ->
      loginWindow.show()
      loginWindow.focus()
```

It works if I set `show:true` and hide it directly using `loginWindow.hide()`. That way I can show it later in the renderer.

![screen shot 2014-07-26 at 11 25 20](https://cloud.githubusercontent.com/assets/431376/3710751/4df43118-14a7-11e4-891a-5791cc430593.png)
![screen shot 2014-07-26 at 11 24 33](https://cloud.githubusercontent.com/assets/431376/3710752/4df485f0-14a7-11e4-8f1e-428599079185.png)

**Comment by murphyrandle** [comment]
Bummer, I'm getting little white caps at the top even when `show:true` is set. Calling `setSize` didn't do the trick either.

![white corners](https://dl.dropboxusercontent.com/u/9297096/white-corners.png)

**Comment by tommoor** [comment]
This definitely doesn't happen if you construct the window with show: true. It also doesn't occur if you resize the window, after a certain point the squared corners disappear.

I think it's important that this rounding remain, as it's the default way that all windows are displayed in OSX and little details like square corners standout as making the app 'feel' not native.

**Comment by tommoor** [comment]
Another very related issue - If you do manage to get the corners to be properly rounded using the hack suggested above then when you move the window between screens of a different DPI the square 'caps' return :frowning:

**Comment by zcbenz** [comment]
Fixed by #1441, an old confusing bug finally solved! :sparkles:
